src/translations/en.json

```diff
@@ -4405,7 +4408,8 @@
       "selected": "{number} selected",
       "failed_to_delete": "Failed to delete",
       "could_not_create": "Could not create backup",
-      "upload_backup": "Upload backup",
+      "upload_backup": "[%key:ui::backup::upload_backup%]",
```

Comment:
Shouldn't this be the other way around?
As we would now invalidate any existing translations?

Reply:
Good point. Fixed.